### ISSUE TITLE
Create win command

### ISSUE THREAD

**Issue by ConnorNelson** [pwncollege/dojo#111]
Program at /opt/pwn.college/win (side note: maybe we should move /opt/pwn.college to /opt/dojo) which will roughly do:

```
/usr/bin/env python

import subprocess

flag = open("/flag").read()
subprocess.run(["/opt/pwn.college/dojo", "submit", flag])
```

Depends on #72 

It will be symlinked into /usr/local/bin to be in the PATH.

Then challenges with a simple win state (most) can simply run the `win` program.

**Comment by zardus** [comment]
Do we really need it to submit the flag automatically? We could just have `win` chmod the flag to be readable.

I like this idea, and it'll open things up to more realistic shellcode (and stuff like making challenges SUID in a way that doesn't let them get at the flag).

**Comment by ConnorNelson** [comment]
We definitely don't *need* to, but there are some (possibly most) challenges where the concept of reading the flag and manually submitting isn't really necessary. You reach some win state, and rather than that being enough, you have to actually copy the flag (slightly more difficult in `desktop`), switch over to the website (or even still use the planned CLI) and submit the flag. This is just a user interaction that doesn't need to happen, it's pretty clear what the win state is when we suddenly `print(flag)`. We also have numerous challenges where the win state is literally calling the `win` function, why not actually have a `win` command.

Obviously we still need a concept of a flag file for challenges where the best you can do is read a file with your elevated privileges.

The only downside I see with this is aesthetic. There is something cool about all of the challenges being closer to a standard jeopardy CTF where you actually capture the **flag**.

A `win` program also gives us other interesting capabilities that are a stretch goal, such as better solution introspection. That `win` program could in theory coredump the parent process, and submit the core alongside the flag for later analysis, snapshot the running container, etc.
